boards/px4/fmu-v6u/src/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ if("${PX4_BOARD_LABEL}" STREQUAL  "bootloader")
 			nuttx_arch # sdio
 			nuttx_drivers # sdio
 			bootloader
+                        px4_layer # px4_platform_console_init
```

Comment:
All of these, linking bootloaders to px4_platform is not valid any more; this has changed - will remove these